internal/logging/database.go

```diff
@@ -0,0 +1,45 @@
+package logging
+
+import (
+	"context"
+	"time"
+
+	"github.com/rs/zerolog"
+	gormlogger "gorm.io/gorm/logger"
+)
+
+type DatabaseLogger struct {
+	zerolog.Logger
+}
+
+func (l DatabaseLogger) LogMode(level gormlogger.LogLevel) gormlogger.Interface {
+	return &DatabaseLogger{l.Logger}
+}
+
+func (l DatabaseLogger) Info(ctx context.Context, format string, v ...interface{}) {
+	l.Logger.Info().Msgf(format, v...)
```

Comment:
How much does this log at the info level?

Reply:
This shouldn't actually log. The only statements I see are trace logs when trace level is enabled